utoipa-gen/src/path/request_body.rs

```diff
@@ -122,11 +122,11 @@ impl ContentTypeResolver for RequestBodyAttr<'_> {}
 
 impl ToTokens for RequestBodyAttr<'_> {
     fn to_tokens(&self, tokens: &mut TokenStream2) {
-        if let Some(ref body_type) = self.content {
-            let property = Property::new(body_type.is_array, &body_type.ty);
+        if let Some(body_type) = &self.content {
+            let property = Property::new(body_type.clone());
```

Comment:
This clone could probably be removed if the type was reference or Cow. :) See above :point_up: